### ISSUE TITLE
代码中未找到CLIP-Guided Enhancement (CGE) module

### ISSUE THREAD

**Issue by XIANGLIU03** [zhangy0822/USER#6]
你好，你们的工作非常棒！在代码在未找到使用CLIP visual encoder的代码，请问要如何使用CGE模块？

**Comment by zhangy0822** [comment]
Thanks for your attention! Please refer to the issue https://github.com/zhangy0822/USER/issues/4#issuecomment-2014270919.

**Comment by zc020126** [comment]
你好，我尝试发现您的代码，但是跑完f30k的数据集后，结果如下：
2024-08-13 03:06:19,855 calculate similarity time: 0.03725838661193848
2024-08-13 03:06:20,003 Image to text: 0.1, 0.3, 0.4, 2491.0, 2482.1
2024-08-13 03:06:20,078 Text to image: 0.1, 0.5, 1.0, 500.0, 500.5
2024-08-13 03:06:20,078 Current rsum is 2.4
请问可能出现的原因是什么呢？

**Comment by Pefect96** [comment]
Please provide the complete training script. In addition, has the loss decreased?

> 你好，我尝试发现您的代码，但是跑完f30k的数据集后，结果如下： 2024-08-13 03:06:19,855 calculate similarity time: 0.03725838661193848 2024-08-13 03:06:20,003 Image to text: 0.1, 0.3, 0.4, 2491.0, 2482.1 2024-08-13 03:06:20,078 Text to image: 0.1, 0.5, 1.0, 500.0, 500.5 2024-08-13 03:06:20,078 Current rsum is 2.4 请问可能出现的原因是什么呢？

**Comment by zhangy0822** [comment]
> 你好，我尝试发现您的代码，但是跑完f30k的数据集后，结果如下： 2024-08-13 03:06:19,855 calculate similarity time: 0.03725838661193848 2024-08-13 03:06:20,003 Image to text: 0.1, 0.3, 0.4, 2491.0, 2482.1 2024-08-13 03:06:20,078 Text to image: 0.1, 0.5, 1.0, 500.0, 500.5 2024-08-13 03:06:20,078 Current rsum is 2.4 请问可能出现的原因是什么呢？

Please provide the complete training script. In addition, has the loss decreased?
